src/css/components/contents.scss

```diff
@@ -0,0 +1,28 @@
+@import '../override_vars.scss';
+@import '../core.scss';
+
+.contents-primary {
```

Comment:
`contents` is the contents of an org or space. The contents styles are used in both the overview card (I should really rename this `home` to reduce confusion...) and the overview panel. I guess it isn't so much a `contents` component as much as common styles between two separate components.

I don't really care where the styles live, however. They don't need to be in this special file. Is there a place you'd prefer them?